storage/drbd/pkg.yaml

```
@@ -21,6 +21,14 @@ steps:
         cp -r /rootfs/ /extensions-validator-rootfs/rootfs
         cp /pkg/manifest.yaml /extensions-validator-rootfs/manifest.yaml
         /extensions-validator validate --rootfs=/extensions-validator-rootfs --pkg-name="${PKG_NAME}"
+    sbom:
+      outputPath: /rootfs/usr/local/share/spdx/drbd.spdx.json
+      version: {{ .DRBD_DRIVER_VERSION }}
+      cpes:
+        - cpe:2.3:a:linbit:drbd8:{{ .DRBD_DRIVER_VERSION }}:*:*:*:*:*:*:*
```

Let's hope they'll adhere to the proper format with version being in the dedicated field

```suggestion
        - cpe:2.3:a:linbit:drbd:{{ .DRBD_DRIVER_VERSION }}:*:*:*:*:*:*:*
        - cpe:2.3:a:linbit:drbd8:{{ .DRBD_DRIVER_VERSION }}:*:*:*:*:*:*:*
```

---

Oh, by the way. Here we copy a package from pkgs, which should have an own SBOM. As we do basically nothing new here, no added components, we should build an SBOM in pkgs (look at NVIDIA non-free modules as an example, their SBOM is also copied in the extensions repo), so that it captures data about what source files we downloaded.

---

Removed.